gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnTaskRunner.java

```diff
@@ -201,6 +201,8 @@ public static void main(String[] args) {
       if (!Strings.isNullOrEmpty(helixInstanceTags)) {
         config = config.withValue(GobblinClusterConfigurationKeys.HELIX_INSTANCE_TAGS_KEY, ConfigValueFactory.fromAnyRef(helixInstanceTags));
       }
+      YarnHelixUtils utils = new YarnHelixUtils();
```

Comment:
Do you need to new the utils class? Can't the updateToken() be made public static?